### PR TITLE
lecturer roles, proposed assignment

### PR DESCRIPTION
**Pull request by fbielejec**

### CODE REVIEW

**Review by wirednkod — APPROVED** [review]

**Review by wirednkod — APPROVED** [review]
Thank you so much for this Filip 🙏🏽 
Looks good to me!

File: syllabus/5-Smart_Contracts/README.md [hunk 1]
```diff
@@ -90,8 +90,18 @@ We will also understand the importance of running such contracts on a trustworth
...
 
 Students will be given a contract with a deliberate vulnerability. They will be tasked with finding it and exploiting. Assignment will be autograded with passing grade when the storage state changes due to the exploit.
 - When? End of Day 1
 - Deadline for submissions: Next morning after day 2 of the lectures (e.g 7am local time as a cutoff).
+
+### Secondary one
+
+The task is to take one of the examples from the hands-on workshop and modify it to meet new criteria.
+The work has to be done both in changing contract's code and in the chain extension implementation (pure Rust, but within FRAME runtime).
+Assignment will be autograded with passing grade when the contract behaves as expected.
```
Comment (wirednkod) [inline syllabus/5-Smart_Contracts/README.md:105]: Lets do it this way: Add a small weight on grading on the 2nd assignment which is for the "over and above and exceptional" students

**Review by wirednkod — APPROVED** [review]
Concerning ###

File: syllabus/5-Smart_Contracts/README.md (hunk 1 above)
Comment (pmikolajczyk41) [reply syllabus/5-Smart_Contracts/README.md:105]: :white_check_mark:

**Pull request merged**